torchmdnet/datasets/comp6.py

```diff
@@ -313,3 +314,87 @@ def len(self):
     def get(self, idx):
         i_subset, i_sample = self.subset_indices[idx]
         return self.subsets[i_subset][i_sample]
+
+
+class COMP6v2(ANIBase):
+    __doc__ = ANIBase.__doc__
```

Comment:
The documentation will appear as in the image 
![image](https://github.com/torchmd/torchmd-net/assets/13015792/85cb98d6-ec2b-41c7-b424-3f57994e0ede)
The COMP6v2 dataset is mentioned in the reference, is this enough?

Reply:
Yes that's not good, I should write some custom documentation for it

Reply:
Updated now